File: lib/models/user_account.dart
class UserAccount {
  final String userId;
  final String pwd;
  final String userName;

  UserAccount({
    required this.userId,
    required this.pwd,
    required this.userName,
  });

  Map<String, dynamic> toMap() {
    return {
      'userId': userId,
      'pwd': pwd,
      'userName': userName,
    };
  }

  factory UserAccount.fromMap(Map<String, dynamic> map) {
    return UserAccount(
      userId: map['userId'] as String,
      pwd: map['pwd'] as String,
      userName: map['userName'] as String,
    );
  }
} 

File: android/app/src/main/java/com/example/capstone_healthcare_app/UserAccount.java
package com.example.capstone_healthcare_app;

public class UserAccount {
    private String userId;
    private String pwd;
    private String userName;

    public UserAccount() {} // Firestore에 필요한 빈 생성자

    public UserAccount(String userId, String pwd, String userName) {
        this.userId = userId;
        this.pwd = pwd;
        this.userName = userName;
    }

    public String getUserId() { return userId; }
    public void setUserId(String userId) { this.userId = userId; }

    public String getPwd() { return pwd; }
    public void setPwd(String pwd) { this.pwd = pwd; }

    public String getUserName() { return userName; }
    public void setUserName(String userName) { this.userName = userName; }
} 